### ISSUE TITLE
Bar chart with numerical horizontal axis

### ISSUE THREAD

**Issue by hlalibe** [gonum/plot#472]
Can't find a way to have a proper numerical values for the horizontal axis in a vertical bar chart.
I can give each bar a label on the horizontal axis, that works,
I can give them an index number 1,2,3,...
But i cannot place them at a specific value, for example first bar at 1, then second bar at 3 so there are no bars showing at 2...
Is this possible ?
Current code with index
```
import (
    "gonum.org/v1/plot/plotter"
    "gonum.org/v1/plot"
    "log"
    "io/ioutil"
    "gonum.org/v1/plot/vg"
)
// Create the plot values and labels.

//create empty slice
var gain_loss plotter.Values
//sequence := make([]string, 199)
sequence := make([]int, 199)
//read rows of gain_loss and append together in slice
for l := 0; l < 198; l++ {
    //gain_loss
    stringup := orders[l][4]
    if err != nil {
        panic(err)
    }
    gain_loss = append(gain_loss, stringup)
}

//read rows of sequence and append together in slice
for m := 0; m < 198; m++ {
    //sequence
    //stringup := orders[m][0]
    //stringup := strconv.FormatFloat(orders[m][0], 'f', 6 ,64)
    stringup := int(orders[m][0])
    if err != nil {
        panic(err)
    }
    sequence = append(sequence, stringup)
}
   
//verticalLabels := []string{"A", "B", "C", "D"}
//verticalLabels = sequence

// Create a vertical BarChart
p1, err := plot.New()
if err != nil {
    log.Panic(err)
}
verticalBarChart, err := plotter.NewBarChart(gain_loss, 0.1*vg.Centimeter)
if err != nil {
    log.Panic(err)
}
p1.Add(verticalBarChart)
//p1.NominalX(verticalLabels...)
err = p1.Save(1600, 300, "profit.png")
if err != nil {
    log.Panic(err)
}

//display on screen
bytes, err := ioutil.ReadFile("profit.png")
display.PNG(bytes)
```

**Comment by ctessum** [comment]
Can you just give the bar at 2 a zero value?
